src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs

```diff
@@ -104,13 +104,39 @@ private async Task NotifyContact(Guid applicationId, int sequenceNo, Cancellatio
                 }
                 else if(sequenceNo == ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO)
                 {
-                    // TO DO: Send an email to notify organisation withdrawal approval to EPAO
                     var lastSubmission = application.ApplyData?.Apply.LatestOrganisationWithdrawalSubmission;
```

Comment:
The code for both cases is almost the same - these could be combined with just the last submission part extracted

Reply:
Updated code